packages/plugin-char/src/Char.ts

```diff
@@ -48,6 +48,9 @@ export class Char<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
         const text = params.text;
         const inline = this.editor.plugins.get(Inline);
         const formats = inline.getCurrentFormats(range);
+        if (params.formats) {
+            formats.append(...params.formats);
+        }
```

Comment:
Looks like a bugfix to me.

Reply:
It is indeed.

Reply:
Not really, just wanted to point it out in case I missed something.